### PR TITLE
fix style of last argument of argument lists

### PR DESCRIPTION
**Pull request by andlaus**
instead of sometimes using a comma for the last item of a list and sometimes not, we now always close lists of items immediately. This does not come with any functional changes, but it makes the coding style a bit more consistent and IMO the code is slightly more readable with this style.

Andreas Lauser &lt;andreas.lauser@mercedes-benz.com&gt;, on behalf of [MBition GmbH](https://mbition.io/).
[Provider Information](https://github.com/mercedes-benz/foss/blob/master/PROVIDER_INFORMATION.md)

### CODE REVIEW

**Comment by andlaus** [comment]
@kayoub5: this is motivated by a comment which you made a while ago :)

File: examples/mksomersaultmodifiedpdx.py [hunk 1]
```diff
@@ -118,11 +118,9 @@ def find_named_object(item_list: List[T], name: str) -> T:
             byte_position=0,
             coded_value=FLIC_FLAC_SID,
             bit_position=None,
-            sdgs=[],
```
[the remark this answers is not in the document]
Comment (kayoub5) [reply examples/mksomersaultmodifiedpdx.py:121]: The idea is if the list or function arguments are multi-line, a trailing comma helps better with diffs

A trailing comma is not needed for a one line list

[the remark this answers is not in the document]
Comment (andlaus) [reply examples/mksomersaultmodifiedpdx.py:121]: yeah. I agree that it makes sense, but my point was that writing a script for "semi-automatic" refactoring for this style is pretty hard... (for the current PR I could basically get away with a bit of regex replacement magic, but I don't see a way to get around a "bracket+string" parser if I have to count the number of arguments, etc.)

[the remark this answers is not in the document]
Comment (andlaus) [reply examples/mksomersaultmodifiedpdx.py:121]: under these constraints, shall we close or merge?

[the remark this answers is not in the document]
Comment (kayoub5) [reply examples/mksomersaultmodifiedpdx.py:121]: just close

**Pull request closed without merging**